.gitlab/e2e_testing_deploy/e2e_deploy.yml

```
@@ -2,18 +2,25 @@
 # e2e_deploy stage
 # Contains jobs which deploy Agent package to testing repsoitories that are used in e2e tests.
 
-.setup_rpm_signing_key: &setup_rpm_signing_key
-  - printf -- "$($CI_PROJECT_DIR/tools/ci/fetch_secret.sh $RPM_GPG_KEY)" | gpg --import --batch
-  - EXIT="${PIPESTATUS[0]}"; if [ $EXIT -ne 0 ]; then echo "Unable to locate credentials needs gitlab runner restart"; exit $EXIT; fi
-  - RPM_SIGNING_PASSPHRASE=$($CI_PROJECT_DIR/tools/ci/fetch_secret.sh $RPM_SIGNING_PASSPHRASE) || exit $?
-
-.setup_apt_signing_key: &setup_apt_signing_key
-  - printf -- "$($CI_PROJECT_DIR/tools/ci/fetch_secret.sh $DEB_GPG_KEY)" | gpg --import --batch
-  - EXIT="${PIPESTATUS[0]}"; if [ $EXIT -ne 0 ]; then echo "Unable to locate credentials needs gitlab runner restart"; exit $EXIT; fi
-  - APT_SIGNING_KEY_PASSPHRASE=$($CI_PROJECT_DIR/tools/ci/fetch_secret.sh $DEB_SIGNING_PASSPHRASE) || exit $?
-
-.setup_signing_keys_package:
-  &setup_signing_keys_package # Set up prod apt repo to get the datadog-signing-keys package
+.setup_signing: &setup_signing
+  - ${CI_PROJECT_DIR}/tools/ci/retry.sh aws s3 cp s3://binaries-ddbuild-io-prod/dd-source/domains/seceng/sit/apps/artifact-security/vault-gpg-client/gpg-vault/dev/gpg-vault-v1.5.1-linux-amd64.tar.gz .
+  - tar -xzf gpg-vault-v1.5.1-linux-amd64.tar.gz
+  - mv gpg-vault-linux-amd64 /usr/bin/gpg-vault-bin
+
+  # Set up gpg shim
+  - echo '#!/bin/bash' > /usr/bin/gpg
```

nit: the canonical shebang for `bash` is `#!/usr/bin/env bash` (for [portability](https://en.wikipedia.org/wiki/Shebang_(Unix)#Portability)):
```suggestion
  - echo '#!/usr/bin/env bash' > /usr/bin/gpg
```
There are several occurrences in the file, but please **feel free to ignore**.